test/unit/header-test.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const fs = require("fs"),
+  chai = require("chai"),
+  sinon = require("sinon"),
+  Header = require("../../app/models/header"),
+  expect = chai.expect;
+
+describe("Header", () => {
+  let header;
+  let db = {
+    find: function () {
+      return;
+    },
+    update: function () {
+      return;
+    },
+    remove: function () {
+      return;
+    }
+  };
+
+  beforeEach(() => {
+
+    header = new Header({}, db);
+
+  });
+
+  it("should call db find with key when calling findByKey", () => {
```

Comment:
@donnapep I didn't add because they are straightforward and there isn't conditional logic in it. I will add some. cheers